gammapy/modeling/parameter.py

```diff
@@ -824,12 +859,16 @@ def __init__(
         min=np.nan,
         max=np.nan,
         error=0,
+        scale_method="scale10",
```

Comment:
AS this class is exposed, it might be great to add a docstring describing it with its parameters (and default values)